anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,156 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from anime_downloader.const import get_random_header
+from selenium.webdriver.common.by import By
+from urllib.parse import urlencode
+from urllib.parse import urlsplit
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from logging import exception
+from sys import platform
+import requests, os
+import click
+import time
+import json
+
+header = get_random_header()
+def test_os():
+    if platform.startswith("linux"):
+        os_name = "linux"
+    elif platform == "darwin":
+        os_name = "darwin"
+    elif platform == "win32":
+        os_name = "win32"
+    return os_name
+gg = test_os()
+if gg.startswith("linux"):
+    dd = '\\'
+    aa = '/'
+elif gg() == 'win32':
+    dd = '/'
+    aa = '\\'
+else:
+    raise exception('This OS is not currently supported.')
+def get_config(): #can't use config because of circular import
+    APP_NAME = 'anime downloader'
+    directory = str(click.get_app_dir(APP_NAME) + aa + 'data').replace(dd, aa)
```

Comment:
Instead of whatever this magic is, use `os.path.join`

Reply:
done